0.5/articles/webview.md

```diff
@@ -0,0 +1,325 @@
+    <link rel="import" href="bower_components/core-toolbar/core-toolbar.html">
+    <link rel="import" href="bower_components/core-icons/core-icons.html">
+    ...
+
+### Main page setup
+
+Create a main page, `src/main/assets/www/index.html` and use an HTML Import to load `elements.html`:
+
+    <!doctype html>
+    <html>
+    <head>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <!-- Note: webcomponents.js is only needed in non-Chrome 36 browsers. It's included here for portability. -->
+      <script src="bower_components/webcomponentsjs/webcomponents.min.js"></script>
+      <link rel="import" href="elements.html">
+      ...
+    </head>
+    <body unresolved fullbleed>
+      ...
+    </body>
+    </html>
+
+It's worth noting that webcomponents.js isn't required for the Chromium 36+ Android WebView. However, if you're creating an app for multiple platforms, it's recommended to include the polyfills for portability.
+
+When all is said and done, your final folder structure should look something like this:
+
+<img src="images/webview/folders.png">
+
+## Configuring app permissions
+
+Now that you have Android L installed and a folder structure in place, it's time to select the correct version of the SDK and tweak WebView permissions.
+
+In `AndroidManifest.xml`, set the minimum and target SDK versions to Android L (or an earlier version if you're relying on the polyfills or Crosswalk). If your app needs access to remote resources (e.g. images, multimedia, JSON endpoints), also request the `android.permission.INTERNET` permission.
+
+**AndroidManifest.xml**:
+
+<pre>
+&lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;
+&lt;manifest ...&gt;
+
+  <b>&lt;uses-sdk
+      android:minSdkVersion=&quot;21&quot;
+      android:targetSdkVersion=&quot;21&quot; /&gt;</b>
+
+  <b>&lt;uses-permission android:name=&quot;android.permission.INTERNET&quot; /&gt;</b>
+
+  &lt;application ...&gt;
+      ...
+&lt;/manifest&gt;
+</pre>
+
+### Tweaking the WebView settings
+
+Out of the box, the WebView disables some features that {{site.project_title}} needs. You can enable them by changing the WebView's `WebSettings`:
+
+- Enable JavaScript.
+- Access to `file://` so HTML Imports can be loaded off `file://` URLs.
+- Local links/redirects act on the WebView (and do not open in the browser).
+
+In **MainActivity.java**, enable the following on your [`WebSettings`](http://developer.android.com/reference/android/webkit/WebSettings.html) object:
+
+<pre>
+public class MyActivity extends Activity {
+
+  private WebView mWebView;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_my);
+
+    mWebView = (WebView) findViewById(R.id.activity_main_webview);
+
+    WebSettings webSettings = mWebView.getSettings();
+
+    // Enable JavaScript.
+    <b>webSettings.setJavaScriptEnabled(true);</b>
```

Comment:
I think this bolding is going to be overridden by the code highlighting.

In the tutorial, we use `class="nocode highlight"` for the bolded text, where `nocode` tells the highlighter to ignore it and the awkwardly-named `highlight` is used to actually increase the text weight (for some reason, I think <b> in a text block defaults to normal weight).